github/ci/prow-deploy/kustom/components/docker-mirror-proxy/pv/patches/JsonRFC6902/deployment.yaml

```diff
@@ -11,3 +11,8 @@
     name: storage
     persistentVolumeClaim:
       claimName: docker-mirror-proxy
+- op: add
```

Comment:
Why is that necessary now?

Reply:
By default the new openshift cluster runs the pods as a non privileged user - the docker-mirror-proxy pod ends up in a error as it tries to access files under /etc

https://github.com/rpardini/docker-registry-proxy/blob/51585675e6543f6e9b81a2b15b15e8f0778119e2/entrypoint.sh#L7

Reply:
Thanks for the explanation!